src/features/skills/components/cardView/card-button/card-button.tsx

```diff
@@ -25,7 +25,7 @@ const CardButton = (props: props) => {
 
     return (
         <button onClick={handleClick} className={styles.container}>
-            <Card title={props.title} image={props.image} description="" />
+            <Card title={props.title} image={props.image} description="" isSkills/>
```

Comment:
@kasumi0 
```suggestion
            <Card description="" isSkills {...props}/>
```

こっちの方が良いかな〜
props継承なので。

Reply:
承知しました！

Reply:
なんか崩れ倒してしまったので、{...props}の順番を最初にしました。